src/components/Book.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Please refrain from using any `eslint-disable` in your code, as it defies the purpose of using linters in the first place.